src/token/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * User: llukac<lukas@lightstreams.io>
+ * Date: 27/11/19 15:15
+ * Copyright 2019 (c) Lightstreams, Granada
+ */
+
+const Web3Wrapper = require('../web3');
+
+module.exports.generateAuthToken = async (web3, { address, tokenBlocksLifespan }) => {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const currentBlock = await Web3Wrapper.getBlockNumber(web3);
+      const expirationBlock = currentBlock + tokenBlocksLifespan;
+
+      const claims = {
+        blockchain: "ETH",
```

Comment:
[Question] Is there any underline reason for this to use "ETH" instead of "PHT" or whatever else string?

Reply:
We use "eth" on backend still given you can use Smart Vault on Ethereum mainnet as well and this Claim attribute is not configurable atm. We should improve this in the future though.